backend/src/test/java/com/redhat/cloud/notifications/routers/sources/SecretUtilsTest.java

```diff
@@ -505,14 +554,15 @@ void updateSecretsForEndpointCreateTest() {
 
         webhookProperties.setBasicAuthentication(basicAuth);
         webhookProperties.setSecretToken(SECRET_TOKEN);
+        webhookProperties.setBearerAuthentication(BEARER_AUTHENTICATION);
 
         endpoint.setProperties(webhookProperties);
         endpoint.setOrgId(orgId);
 
         // Set up the mock calls for the "create" calls from the REST Client. Make sure we return the basic
         // authentication's ID first, and the secret token's ID second, since we are expecting a successful create
         // operation.
```

Comment:
```suggestion
        // Set up the mock calls for the "create" calls from the REST Client.
        // Make sure we return the basic authentication's ID first, the secret
        // token's ID second and the bearer authentication's ID third, since we
        // are expecting a successful create operation.
```